src/sentry/integrations/slack/utils/escape.py

```diff
@@ -1,8 +1,12 @@
+from __future__ import annotations
+
 # TODO(scttcper): Might need to handle "*" bold, and "_" italics
 translator = str.maketrans({"&": "&amp;", "<": "&lt;", ">": "&gt;"})
 
 
-def escape_slack_text(txt: str) -> str:
+def escape_slack_text(txt: str | None) -> str:
+    if not txt:
+        return ""
```

Comment:
Should this be under the functions docstring?

Reply:
fixed